pkg/querier/queryrange/codec.go

```diff
@@ -1137,9 +1173,9 @@ func paramsFromRequest(req queryrangebase.Request) (logql.Params, error) {
 		return &paramsSeriesWrapper{
 			LokiSeriesRequest: r,
 		}, nil
-	case *LokiLabelNamesRequest:
+	case *LabelRequest:
 		return &paramsLabelNamesWrapper{
-			LokiLabelNamesRequest: r,
+			LabelRequest: r,
```

Comment:
I think we can rename `paramsLabelNamesWrapper` to `paramsLabelWrapper` to follow the new pattern you're introducing